cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,18 +1,29 @@
+import {MenuContentPage,ShopingCartPage, ProductListPage,SumaryPage,LoginPage,AddressSetpPage, ShippingStepsPage,PaymentPage} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const shopingCartPage = new ShopingCartPage();
+const sumaryPage = new SumaryPage();
+const loginPage = new LoginPage();
+const addressSetpPage = new AddressSetpPage();
+const shippingStepsPage = new ShippingStepsPage();
+const paymentPage = new PaymentPage();
+const productListPage = new ProductListPage();
```

Comment:
productsListPage*

Reply:
We have a typo as well on shopping-cart.page.ts and we are missing the OrderSummaryPage but these are the names that we asked to place
![Screen Shot 2022-05-10 at 5 46 57 PM](https://user-images.githubusercontent.com/36927617/167735877-61b6083f-2f76-4782-bef8-6dbe132dd09f.png)